CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+
+- Typescript support (made with ❤ by UFCG).
```

Comment:
maybe is good add which components already is in ts, because keep the track of the entropy easier.

Reply:
It is in the ```Changed` section @ArthurSampaio